universal-application-tool-0.0.1/app/services/cloud/aws/SimpleStorage.java

```diff
@@ -65,10 +66,13 @@ public SimpleStorage(
         });
   }
 
-  @Override
   public URL getPresignedUrl(String key) {
-    GetObjectRequest getObjectRequest = GetObjectRequest.builder().key(key).bucket(bucket).build();
+    return getPresignedUrl(key, Optional.empty());
```

Comment:
suggestion (blocking): please add an issue and TODO for supporting original filename for AWS uploads